### PR TITLE
Adding FS TC 83574328

### PR DESCRIPTION
**Pull request by julpark-rh**
Automation - T2 - CEPH-83574328 - Modify an existing permissions on the ceph user and verify the changes
log:https://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1VX7D2

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/clients/ceph_auth_caps_modifying_permissions.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+                        raise CommandFailed(
+                            f"Not Expected permission for {daemon1}, it should be {perm1}"
+                        )
+                    if output3["caps"][daemon2] != perm2.strip("'"):
+                        raise CommandFailed(
+                            f"Not Expected permission for {daemon2}, it should be {perm2}"
+                        )
+                    if output3["caps"][daemon3] != perm3.strip("'"):
+                        raise CommandFailed(
+                            f"Not Expected permission for {daemon3}, it should be {perm3}"
```
Comment (AmarnatReddy) [inline tests/cephfs/clients/ceph_auth_caps_modifying_permissions.py:127]: Along with this, can we also do some IO operations based on the Permission set.
we can mount the cephfs using the user and do IO operations

Comment (julpark-rh) [reply tests/cephfs/clients/ceph_auth_caps_modifying_permissions.py:144]: I have added IO operation at the beginning permission modifying.

Comment (hkadam134) [reply tests/cephfs/clients/ceph_auth_caps_modifying_permissions.py:144]: @julpark-rh - After every permission change, please run IO's and validate the permissions accordingly.

**Comment by julpark-rh** [comment]
https://github.com/red-hat-storage/cephci/pull/2551

**Pull request closed without merging**